From 2a52eaaf164a162b32fc82e2b0deeca9b7d46d60 Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Fri, 21 Dec 2012 13:31:18 +0100
Subject: [PATCH 22/53] test_fragments: increase message size since it
 succeeds

We get the following error:

  test_fragments.c  3 BROK : Send a message that exceeds assoc frag
                             point error:30000 errno:0

This means, it just succeeds, so increase the message length!

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
---
 src/func_tests/test_fragments.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/func_tests/test_fragments.c b/src/func_tests/test_fragments.c
index 15aba6f..973d485 100644
--- a/src/func_tests/test_fragments.c
+++ b/src/func_tests/test_fragments.c
@@ -224,7 +224,7 @@ main(int argc, char *argv[])
 	/* Try to send a messsage that exceeds association fragmentation point
 	 * and verify that it fails.
 	 */
-	msg_len = 30000;
+	msg_len = 100000;
 	msg_buf = test_build_msg(msg_len);
 	outmessage.msg_iov->iov_base = msg_buf;
 	outmessage.msg_iov->iov_len = msg_len;
-- 
1.7.11.7

